README.md

```diff
@@ -290,6 +290,7 @@ Individual commands may also set tool-specific targets:
 
 - `vsim`
 - `vcs`
+- `xm`
```

Comment:
Can we change this name to `xcelium`? This applies to all uses below (command name, docs, ...)

Reply:
Done